lighthouse-core/gather/driver.js

```diff
@@ -840,6 +840,7 @@ class Driver {
     // happen _after_ onload: https://crbug.com/768961
     this.sendCommand('Page.enable');
     this.sendCommand('Emulation.setScriptExecutionDisabled', {value: disableJS});
+    this.setNextProtocolTimeout(30 * 1000);
```

Comment:
> These can 'race' and that's OK.

apparently not :)

Feels like we should `await` `Page.enable` and `Emulation.setScriptExecutionDisabled` (either they're short and it's no big deal or they're long and we need to account for them) but we can save that for another day.